app/src/main/java/fr/free/nrw/commons/upload/UploadPresenter.java

```diff
@@ -35,6 +35,8 @@ public class UploadPresenter implements UploadContract.UserActionListener {
     public static final String COUNTER_OF_CONSECUTIVE_UPLOADS_WITHOUT_COORDINATES
         = "number_of_consecutive_uploads_without_coordinates";
 
+    public static final int CONSECUTIVE_UPLOADS_REMINDER_THRESHOLD = 10;
```

Comment:
Isn't this threshold already defined elsewhere? Better avoid defining the same in several places.

Reply:
Sorry I did not realize it was actually hard-coded 😅
Defining a static final like this is much better.
I would suggest naming it CONSECUTIVE_UPLOADS_WITHOUT_COORDINATES_REMINDER_THRESHOLD for extra clarity. Long names are not considered a problem in Java.

Reply:
Sounds good! I've renamed the variable